recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
```

Comment:
I found it :)

if I directly install by

`conda install -c conda-forge netcdf4`


I get 

`hdf4                      4.2.11                        4    conda-forge`

if I do the upgrade from default to conda-forge I have only

`hdf4                      4.2.11                        0    conda-forge`

If I do after update of netCDF4 another one for hdf4

` conda install -c conda-forge hdf4`

the issue is gone